src/main.ts

```diff
@@ -49,6 +49,18 @@ const main = async () => {
     }
 }
 
+!oneOff && process.on("uncaughtException", async function(e) {
```

Comment:
I think maybe we want to keep the uncaughtException event listener for oneOff, just not have it trigger the recursive setTimeout (i.e., only log). The reason I'd removed it is I thought the process would stay running indefinitely but I don't think that's the case (this was my change, not yours, so I can change it on the one-offs branch later if you'd like)

Reply:
Yes we can add uncaughtException support for oneOff, i can help directly on this pr.